programmer-humour/script.js

```diff
@@ -0,0 +1,26 @@
+async function fetchComic(){
+    try {
+        const response = await fetch('https://xkcd.now.sh/?comic=latest');
+        const data = await response.json();
```

Comment:
I don't think you need await in front of response.json() here. You already waited for the fetch().